networking_generic_switch/devices/netmiko_devices/dell.py

```diff
@@ -50,7 +50,7 @@ class DellNos(netmiko_devices.NetmikoSwitch):
     QUERY_PORT = (
         'end', # Need to briefly leave configuration mode.
         'show interfaces switchport {port} | grep ^U',
-        'conf', # Re-enter configuration mode.
+        'configure', # Re-enter configuration mode.
```

Comment:
Ah! I saw 'invalid input' error when sending commands to the switch, and it always failed at this "conf" command. so I thought it was because of the abbreviation, but it was not. I'll change it back.